homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -22,19 +22,21 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up the LG platform."""
     if discovery_info is not None:
-        add_entities([LGDevice(discovery_info)], True)
-
+        async_add_entities([LGDevice(discovery_info)], True)
+    if config.get("host") and config.get("port"):
```

Comment:
The correct fix is to define a `PLATFORM_SCHEMA` in the module here that requires host and port. See other platforms for examples.